internal/e2e/reference_types_directive/reference_types.spec.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="zone.js" />
+/// <reference types="hammerjs" />
+
+// The jasmine types are resolved because the tsconfig.json
+// for this compilation includes "types": ["jasmine"].
+// This file will fail to compile if that mechanism of including types
+// is broken
+
+describe('reference types directive resolution', () => {
+  it('should resolve zone.js types from node_modules/zone.js/dist/zone.js.d.ts', () => {
+    // The type of Zone should resolve or this will fail to compile
+    // but since zone is not part of type test, typeof Zone should be
+    // equal to 'undefined'
+    expect(typeof Zone).toEqual('undefined');
```

Comment:
it's pretty confusing - it looks like we're asserting that zone is broken, despite your comment.
Can we have any positive test first? Even just
let zone: Zone;
or something that would fail if the reference weren't there?

Reply:
good call. `let zone: Zone;` is less confusing